classroom.rb

```diff
@@ -0,0 +1,15 @@
+class Classroom
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
+
+  def add_student(student)
+    @students << film
```

Comment:
- [ ] Hi you did a great job but you're supposed to append the student variable to your @students instance variable 
`@students << student`